src/server.js

```diff
@@ -21,6 +22,7 @@ app.use(express.json());
 
 // routes
 app.use('/api/post', postRouter);
+app.use('/api/group', groupRouter);
```

Comment:
This is a preference, but it seems to be a more popular convention to pluralize. the endpoints, i.e. `/api/groups`. Because there are multiple "group" resources, this makes more sense to me.

I accidentally left `posts` as `/api/post` but have a PR to pluralize.

Happy to discuss this and we can all agree on it going forward, but definitely want to be consistent.

Reply:
Agree with you.